### ISSUE TITLE
edit-pr-comment functionality doesn't create new comment for different projects inside repo

### ISSUE THREAD

**Issue by miguel-botelho** [pulumi/actions#754]
### What happened?

If there are two (or more) Github actions that execute the same Pulumi command on different projects, the Action will output both jobs on the same Github comment, given that the edit-pr-comment flag is set to true.

I believe the issue resides on the fact that the current Github comment only specifies the stack for which it ran the command, not the project name. Adding the project name would make the comment unique per project + stack (right now, it only looks for the stack: https://github.com/pulumi/actions/blob/2e6eedbfaf1d031dd23a2d289c64e048900598ee/src/libs/pr.ts#L18 and https://github.com/pulumi/actions/blob/2e6eedbfaf1d031dd23a2d289c64e048900598ee/src/libs/pr.ts#L55)

### Steps to reproduce

- Have two jobs in the same Github action, such as:
```
name: Refresh Production

env:
  EC_API_KEY: ${{ secrets.EC_API_KEY }}
  GITHUB_TOKEN: ${{ secrets.PULUMI_GITHUB_TOKEN }}
  PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}

on: workflow_dispatch

jobs:
  pulumi-infra:
    name: Pulumi Infra Production
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
          node-version: 18.x
      - uses: google-github-actions/auth@v0
        with:
          credentials_json: ${{ secrets.PRODUCTION_GCP_CREDENTIALS }}
      - run: npm install
      - uses: pulumi/actions@v3
        with:
          work-dir: ./infra/
          command: refresh
          stack-name: production
          diff: true
  pulumi-gcp-metrics:
    name: Pulumi GCP Metrics Production
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
          node-version: 18.x
      - uses: google-github-actions/auth@v0
        with:
          credentials_json: ${{ secrets.PRODUCTION_GCP_CREDENTIALS }}
      - run: npm install
      - uses: pulumi/actions@v3
        with:
          work-dir: ./gcp_metrics/
          command: refresh
          stack-name: production
          diff: true
```
- Run the action

### Expected Behavior

Two comments should be outputted, one for each job.

### Actual Behavior

Only one comment is outputted, as the second action that was ran edits the original comment.

### Additional context

I can open a PR, as I believe the issue should be more or less simple. I was thinking of adding the work-dir or the project name in the comment itself, and the comment would then be more unique.

Basically going from this:
<img width="922" alt="image" src="https://user-images.githubusercontent.com/9268166/195578689-489a7503-d8ba-453f-82c6-2435f6e9ef1b.png">

To this:
<img width="918" alt="image" src="https://user-images.githubusercontent.com/9268166/195578876-4adcd903-357b-4e19-840c-40fc7eb58aa4.png">

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by RobbieMcKinstry** [comment]
Hi @miguel-botelho . Thanks for opening this issue. I think this is a duplicate of https://github.com/pulumi/actions/issues/563
Nevertheless, your solution sounds correct to me. We'd be happy to accept a PR!

>  I was thinking of adding the work-dir or the project name in the comment itself

I think I'd prefer the project name over the work-dir. The way I think about it, `project name` is a logical distinction Pulumi makes between programs, and `work-dir` is an implementation detail that lets users specify multiple projects in a monorepo. Off the top of my head, I don't think it's possible that using `project name` could introduce a conflict -- organization + project name + stack name should uniquely identity the source of the run.

I'm going to close this issue as a dup, but I'd be happy to reopen if you disagree. Nevertheless, we'd still love a PR to address the problem and close https://github.com/pulumi/actions/issues/563 :-)

**Comment by miguel-botelho** [comment]
Thanks @RobbieMcKinstry ! I'll tackle it in #563 then, and sorry for not catching it while looking in the list of open issues 😅
